### ISSUE TITLE
Repeated focus event listener in a11y.mjs

### ISSUE THREAD

**Issue by mrleemon** [nolimits4web/swiper#8139]
**Description**

There is a duplicate line of code in the `a11y` module where the same event listener is attached twice to the swiper element. This is redundant and should be cleaned up to avoid unnecessary execution or potential side effects.

**Location**

File: src/modules/a11y/a11y.mjs Lines: [354-355](https://github.com/nolimits4web/swiper/blob/ec4977b629c0823173e7b8bde7feb040a9fc4ff3/src/modules/a11y/a11y.mjs#L354C5-L355C60)

**Code Snippet**

```
swiper.el.addEventListener('focus', handleFocus, true);
swiper.el.addEventListener('focus', handleFocus, true);
```

**Expected Behavior**

The focus event listener should only be added once.

**Actual Behavior**

The focus event listener is added twice, which means `handleFocus` will be triggered twice for every focus event on the swiper element.

**Suggested Fix**

Remove one of the duplicate lines.
